### PR TITLE
Civ 3760 uncloaking without notice make order

### PR DESCRIPTION
**Pull request by Azam-Hmcts**
**Before creating a pull request make sure that:**

- [X] commit messages are meaningful and follow good commit message guidelines
- [X] README and other documentation has been updated / added (if needed)
- [X] tests have been updated / new tests has been added (if needed)
- [X] Screenshot attached for the Evidence of manual testing
- [x] Functional, Smoke and API tests have been run locally
- [X] Full gradle build and run tests with coverage have been completed (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-3760

### Change description ###

When a judge uncloaks the notification when making an order and approving the order, the state of the application is updated to either ORDER_MADE, AWAITING_DIRECTIONS_ORDER_DOCS or APPLICATION_DISMISSED.  If the "Make application visible to all parties" is checked and order submitted (regardless of the order type), this will trigger the order notification to both parties and no additional payment is required.  Both the applicant and respondent will then be able to see the application tab and the details and the order on the application.

### Testing branch and instruction ###

CIV-3760-TestBranch

### -----Manual testing - Cloaked Application---- ###

**Application Creation**

image

**Judges Make order With cloaked applicaotin**

image

**Final State**
image

** Only one email is sent **

image

### -----Manual testing - UnCloaked Application---- ###

image

**uncloaked**
image

**Final State and emails sent**
image

image

**Does this PR introduce a breaking change?** (check one with "x")

### Fucntional-TEST ###

image

### Smoke-TEST ###

image

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -52,9 +52,9 @@ withPipeline(type, product, component) {
     disableLegacyDeployment()
     enableAksStagingDeployment()
 
-    onPR {
```
Comment (karthick-mohanasundaram-HMCTS) [inline Jenkinsfile_CNP:55]: could we revert back this indentation changes?

File: e2e/tests/ui_tests/smoke_test.js [hunk 2]
```diff
@@ -55,6 +55,6 @@ Scenario('GA for 1v1- respond to application - Request more information', async
   console.log('Responded to Judge Additional Information on case: ' + childCaseNum());
 }).retry(0);
 
-AfterSuite(async ({api}) => {
-  await api.cleanUp();
-});
+// AfterSuite(async ({api}) => {
```
Comment (karthick-mohanasundaram-HMCTS) [inline e2e/tests/ui_tests/smoke_test.js:58]: @Azam-Hmcts  is this intentional?

Comment (Azam-Hmcts) [reply e2e/tests/ui_tests/smoke_test.js:58]: Yeah, it was. But good eye. Its not more needed.

**Review by karthick-mohanasundaram-HMCTS — DISMISSED** [review]

**Review by smitatalawadekar-hmcts — APPROVED** [review]

**Review by MMNycz — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -7,10 +7,10 @@
     "test": "npx codeceptjs run --steps",
     "test:smoke": "MOCHAWESOME_REPORTFILENAME=smoke REPORT_DIR=test-results/smokeTest REPORT_FILE=test-results/smokeTest/results.xml npx codeceptjs run --grep @smoke-tests --reporter mocha-multi --verbose",
     "test:ga-smoke": "MOCHAWESOME_REPORTFILENAME=ga-smoke REPORT_DIR=test-results/smokeTest REPORT_FILE=test-results/smokeTest/results.xml npx codeceptjs run --grep @ga-smoke-tests --reporter mocha-multi --verbose",
-    "test:api": "MOCHAWESOME_REPORTFILENAME=api npx codeceptjs run --grep @api-tests --reporter mocha-multi --verbose",
+    "test:api": "npx codeceptjs run --grep @api-tests --reporter mocha-multi --verbose",
     "test:e2e": "MOCHAWESOME_REPORTFILENAME=e2e npx codeceptjs run --grep @e2e-tests --reporter mocha-multi --verbose",
     "test:e2e-multiparty": "MOCHAWESOME_REPORTFILENAME=e2e-multiparty npx codeceptjs run --grep @multiparty-e2e-tests --reporter mocha-multi --verbose",
-    "test:functional": "MOCHAWESOME_REPORTFILENAME=functional npx codeceptjs run --grep '@e2e-tests|@multiparty-e2e-tests' --reporter mocha-multi --verbose",
```
Comment (deepthidoppalapudihmcts) [inline package.json:13]: Are we not supposed to revert the changes to the package.json after testing in local?

Comment (Azam-Hmcts) [reply package.json]: Removed local Changes

**Review by deepthidoppalapudihmcts — APPROVED** [review]

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]

**Review by MMNycz — APPROVED** [review]
